ros2topic/test/test_info.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from argparse import Namespace
+from contextlib import contextmanager
+from io import StringIO
+import sys
+
+from ros2topic.verb.info import InfoVerb
+
+NODE_NAME = 'bar'
```

Comment:
This variable doesn't seem to be used anywhere?

Reply:
Yes, it is unused, deleted. Interesting that flake8 did not detect it. 

Now why this code existed at some point and addressing the comment on the value of the test.  So this test is not super valuable, but still useful.  It will fail if node.count_subscribers() fails.  That's useful.  Also having a test that documents expected output is also useful.  It is a reminder that changing output format will break users who do text parsing.

I tried to come with more complicated test (`NODE_NAME` is a leftover), but creating publisher, subscriber and calling info verb will require 2 additional threads + syncronization, and it was just too much for such trivial thing(or I just don't know how to do it simpier). Better testing is available in [PR in rclpy](https://github.com/ros2/rclpy/pull/183).